ci-operator/step-registry/openshift/manifests/external/openshift-manifests-external-ref.yaml

```diff
@@ -0,0 +1,20 @@
+ref:
+  as: openshift-manifests-external
+  from: tools
+  commands: openshift-manifests-external-commands.sh
+  resources:
+    requests:
+      cpu: 100m
+      memory: 100Mi
+  env:
+  - name: EXTERNAL_MANIFESTS_URL
+    default: ""
+    documentation: URL to fetch manifest file from
+  - name: EXTERNAL_MANIFESTS_SHA256_HASH
+    default: ""
+    documentation: Expected manifest file sha256 hash
+  - name: EXTERNAL_MANIFESTS_POST_INSTALL
+    default: "false"
+    documentation: If set to "true" this would place them in SHARED_DIR to be picked up by installed. Any other value would use `oc apply` (using KUBECONFIG env var) to apply the manifest on live cluster.
```

Comment:
I think the sense is backwards here, with `EXTERNAL_MANIFESTS_POST_INSTALL=true` giving us `oc apply`, and everything else giving `SHARED_DIR` handling.

Reply:
Right, fixed in ad233fd49a